selection.tf

```diff
@@ -16,6 +16,9 @@ resource "aws_backup_selection" "ab_selection" {
       value = lookup(selection_tag.value, "value", null)
     }
   }
+
+  not_resources = []
```

Comment:
@tchernomax thanks for the PR. It seems this new feature [was introduced](https://github.com/hashicorp/terraform-provider-aws/releases#:~:text=resource/aws_backup_selection) in the AWS provider version 3.72.0.

I see it fixes the issue but I would a moe dynamic solution, similar to the resources attribute:

```
resources = lookup(element(local.selections, count.index), "resources", null)
not_resources = lookup(element(local.selections, count.index), "not_resources", null)
```

And include the following to the locals:

```
  # Selection
  selection = var.selection_name == null ? [] : [
    {
      name           = var.selection_name
      resources      = var.selection_resources
      not_reesouces  = var.selection_not_resources
      conditions     = var.selection_conditions
      selection_tags = var.selection_tags
    }
  ]
```

And something similar for `condition`. I'm going to merge your code and implement a dynamic behaviour.